### ISSUE TITLE
worcs or WORCS?

### ISSUE THREAD

**Issue by brandmaier** [cjvanlissa/worcs#44]
The package is currently referred to as worcs (all lowercase as it is common with R packages). In the manuscript title, the approach is referred to as WORCS (all uppercase). Is this difference intended?

**Comment by cjvanlissa** [comment]
Yes, the name is an acronym, and as such, should be in upper case. The package name is in lower case, which is the convention (https://alexandereckert.com/post/r-package-names/r-package-names/), I believe because before you install a package, you don't know which letters are capitalized. This distinction also helps clarify when we're talking about the workflow vs the software.

**Comment by aaronpeikert** [comment]
Maybe to make this distinction more pronounced format the package as code, so it is WORCS (the approach) vs `worcs` (the package)?

**Comment by cjvanlissa** [comment]
Yes, in the package I've tried to do that. The paper doens't really mention the package much yet, so I will check
